### ISSUE TITLE
[Question] High error rate after training - why?

### ISSUE THREAD

**Issue by jovargas** [OCR-D/ocrd-train#13]
I'm training with a new set of two fonts and the goal is to use Tesseract to analyze individual chars (only capital letters and numbers), not entire words, but the results are far from being decent and it looks like I'm doing something wrong. Tesseract and Leptonica are installed by the scripts.

Inspired by the test set provided in this repo, I created these tif files with their correct gt.txt's:

From original binarized chars:
![image](https://user-images.githubusercontent.com/20908811/42037385-a87af726-7abe-11e8-9cb6-03da81dd0459.png)

From two TTFs to TIF images with random text:
![image](https://user-images.githubusercontent.com/20908811/42037441-ce05fd7e-7abe-11e8-9c61-f0c55e0d1c5d.png)
![image](https://user-images.githubusercontent.com/20908811/42037476-df968478-7abe-11e8-8339-70c5912c79a9.png)

At the end of the data creation process I have 1869 mixed text lines.

First I ran the makefile with 10000 iterations as default, but the best error rate was high.
I thought it was a matter of needing more iterations, so I changed to 30000 but nothing got better. The following image shows the results:
![image](https://user-images.githubusercontent.com/20908811/42038282-f5d565f4-7ac0-11e8-958e-1757dfc24abd.png)

Sometimes I can see the char train increasing instead of getting lower.
What am I missing here? Do I need more training data? Is my initial data not following any important concept?
I'd appreciate any help!

**Comment by kba** [comment]
It's probably best to [ask on the tesseract-ocr mailing list](https://groups.google.com/forum/#!forum/tesseract-ocr), we're only providing a frontend, your problem is caused by tesseract itself.

If it is indeed a problem with how we're laying out directories or defaults given in the Makefile, we'll be happy about a PR.

**Comment by kba** [comment]
https://groups.google.com/forum/#!topic/tesseract-ocr/qiQLb2-_QlE

**Comment by jovargas** [comment]
Well, it was me who opened that thread, hehe. 
Thank you @kba!
